apps/etterlatte-behandling/src/main/kotlin/grunnlagsendring/GrunnlagsendringshendelseService.kt

```diff
@@ -333,6 +334,30 @@ class GrunnlagsendringshendelseService(
             }.map { it.first }
     }
 
+    fun opprettDoedshendelseForPerson(grunnlagsendringshendelse: Grunnlagsendringshendelse): OppgaveIntern? {
```

Comment:
Mulig vi må ser på hele flyten senere for for meg så blir det kanskje ikke så nødvendig med disse sjekkene her i tillegg til de som gjøres av jobben før dette? mtp på eksisterende

Reply:
Endte opp med en større omskriving her, kanskje greiest med en gjennomgang her. 
[703329e](https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/3927/commits/703329e94dde12c711eac56f47fe36e596dc9739)